From 1a42010ef57549a7e21edcfb2d9230fd6561920c Mon Sep 17 00:00:00 2001
From: Yan Markman <ymarkman@marvell.com>
Date: Fri, 31 Oct 2025 13:08:50 +0200
Subject: [PATCH 1/1] platform/marvell-prest: move PLATFORM_CN9131 fit_addr up
 trixie

On TRIXIE
'initrd' loaded into 0x6000000 with size exceeding the address 0x8000000.
This overlaps with FIT-image loaded into fdt_addr=0x8000000.
FIX: Move loading address up to fdt_addr=0x9000000

Signed-off-by: Yan Markman <ymarkman@marvell.com>
---
 platform/marvell-prestera/platform_arm64.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/platform/marvell-prestera/platform_arm64.conf b/platform/marvell-prestera/platform_arm64.conf
index b6abc625a..978ddd3f4 100644
--- a/platform/marvell-prestera/platform_arm64.conf
+++ b/platform/marvell-prestera/platform_arm64.conf
@@ -66,7 +66,7 @@ elif [ $PLATFORM_7215_A1 -eq 1 ]; then
     demo_part=2
 elif [ $PLATFORM_CN9131 -eq 1 ]; then
     fdt_addr=0x1000000
-    fit_addr=0x8000000
+    fit_addr=0x9000000
     initrd_addr=0x2000000
     FW_ENV_DEFAULT='/dev/mtd1 0x1F0000 0x10000 0x10000'
 else
-- 
2.34.1

